remix-gql-tailwind/app/components/RepoNavigation/RepoNavigation.tsx

```diff
@@ -1,7 +1,7 @@
 import { GitBranchIcon } from '../Icons';
 import { Link } from '@remix-run/react';
 import { useRepo } from '../../context/RepoContext';
-import * as styles from './FileExplorerNav.classNames';
+import * as styles from './RepoNavigation.classNames';
 import { removePathPart } from '../../lib/pathUtils';
 
```

Comment:
While things still render fine on the page, it might be worth explicitly importing React. The linter is complaining about it.

<img width="706" alt="image" src="https://user-images.githubusercontent.com/2487968/183109177-aac92982-9a4b-4a2c-af81-9567fbf390c8.png">

Reply:
okay,good...just added it